src/python/bot/tasks/testcase_manager.py

```diff
@@ -487,37 +490,56 @@ def run_testcase_and_return_result_in_queue(crash_queue,
                    'run_testcase_and_return_result_in_queue.')
 
 
-def test_for_crash_with_retries(testcase,
-                                testcase_path,
-                                test_timeout,
-                                http_flag=False,
-                                compare_crash=True):
-  """Test for a crash and return crash parameters like crash type, crash state,
-  crash stacktrace, etc."""
-  # Cleanup any existing application instances and user profile directories.
-  # Cleaning up temp clears user profile directories and should be done before
-  # calling |get_command_line_for_application| call since that creates
-  # dependencies in the profile folder.
-  process_handler.terminate_stale_application_instances()
-  shell.clear_temp_directory()
-
-  app_directory = environment.get_value('APP_DIR')
-  command = get_command_line_for_application(
-      testcase_path, needs_http=http_flag)
-  crash_retries = environment.get_value('CRASH_RETRIES')
-  flaky_stacktrace = testcase.flaky_stack
-  warmup_timeout = environment.get_value('WARMUP_TIMEOUT')
+class TestcaseRunner(object):
+  """Testcase runner."""
+
+  def __init__(self,
+               fuzzer_name,
+               testcase_path,
+               test_timeout,
+               gestures,
+               needs_http=False):
+    self._testcase_path = testcase_path
```

Comment:
super nit: the order here matches neither the order of arguments above nor the alphabetical order

Reply:
Fixed to match order of arguments.